README.md

```diff
@@ -6,7 +6,9 @@
 
 <!-- # Connaisseur -->
 
-Connaisseur is an admission controller for Kubernetes that integrates Image Signature Verification into a cluster, as a means to ensure that only valid images are being deployed.
+Connaisseur is an admission controller for Kubernetes that integrates Image Signature Verification and Trust Pinning into a cluster, as a means to ensure that only valid images are being deployed.
+
+🚨 🚨 We have added experimental support for [Cosign](https://github.com/sigstore/cosign) from the [Sigstore Project](https://sigstore.dev/). You can find instructions [here](setup/COSIGN.md). 🚨 🚨
```

Comment:
Maybe add here that experimental does mean that it is not part of our semantic versioning guarantees and we take the liberty to remove it with any version at any time without incrementing major

Reply:
Will do that on the actual `COSIGN.md`

Reply:
done